### ISSUE TITLE
Methods to delete elements from problem

### ISSUE THREAD

**Issue by dreinon** [tpaviot/ProcessScheduler#61]
It would be nice if there could be methods to delete constraints, objectives, resources, and tasks.
This would work if we wanted to iterate in solving the same problem but without some elements, so we wouldn't need to create a new problem and add all wanted elements again.

**Comment by tpaviot** [comment]
Right, so far there's not any editing possibility (add/modify/remove).
